complementing_a_strand_of_dna/reverse_complement_dna_string.py

```diff
@@ -0,0 +1,29 @@
+# take in a string of nucleobases and print the reverse complement
+import argparse as ap
+
+# store the complements of each base
+COMPLEMENTS = {
+          'A': 'T'
+        , 'T': 'A'
+        , 'C': 'G'
+        , 'G': 'C'
+}
+
+if __name__ == "__main__":
+    # set up argument parser
+    parser = ap.ArgumentParser(description = "parse a file containing a dna string and return the reverse complement of it")
+    # add input parsers
+    parser.add_argument('-f', help = "the relative path of the input file", type = ap.FileType('r'), required = True)
+    # parse arguments and collect variables
+    args = parser.parse_args()
+    raw_data = args.f
+
+    for line in raw_data.read().splitlines():
```

Comment:
note I'm not doing any input validation here...may do it later on, we'll see how things go